game/scripts/vscripts/common/init.lua

```diff
@@ -13,5 +13,5 @@ require("common/personal_items_cooldown")
 require("common/fast_items")
 require("common/block_holding_wards")
 require("common/patreons_game_perk/patreon_game_perk")
-
+require("common/AutoTeamPatreon/index")
```

Comment:
That would be more consistent with other scripts
```suggestion
require("common/auto_team")

```